python/ray/autoscaler/aws/cloudwatch/prometheus_ray_up.sh

```diff
@@ -0,0 +1,33 @@
+###########################################################################################################
+#This bash script is meant to support prometheus metrics collection via cloudwatch agent.
+#You can view the ray metrics being collected through AWS CloudWatch console.
+###########################################################################################################
+
+read -r -p "please specify your CLUSTER_CONFIG_FILE to start a cluster" AUTOSCALER_CONFIG_FILE
+CLUSTER_NAME=$(grep 'cluster_name' "$AUTOSCALER_CONFIG_FILE"| cut -d':' -f2 | xargs)
+
+# ray up to start a new cluster
+ray up "$AUTOSCALER_CONFIG_FILE"
+
+if [ $? -eq 0 ]; then
+  # download the latest prometheus
+  ray exec "$AUTOSCALER_CONFIG_FILE" "sudo wget https://github.com/prometheus/prometheus/releases/download/v2.25.0/prometheus-2.25.0.linux-amd64.tar.gz -P /opt"
```

Comment:
Let's add a `#TODO` comment here to remove the version number hardcoding from this script and replace it with a more maintainable method that retrieves and installs the latest version of prometheus passing both CloudWatch and Ray integration tests.